components/datepicker/WeekDay.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const WeekDay = ({ weekday, className, localeUtils, locale, size }) => {
+  const weekdayName = localeUtils.formatWeekdayLong(weekday, locale);
+
+  return (
+    <div className={className} title={weekdayName}>
+      {size === 'large' ? weekdayName.slice(0, 2) : weekdayName.slice(0, 1)}
```

Comment:
PP could write this a bit shorter (and IMO I think it's easier to read) by putting the if around the part that's different:
`{weekdayName.slice(0, size === 'large' ? 2 : 1)}`